utils/.config.template.toml

```
@@ -13,7 +13,7 @@ random = { optional = true, options = [true,
     false,
 ], default = false, type = "bool", explanation = "If set to no, it will ask you a thread link to extract the thread, if yes it will randomize it. Default: 'False'", example = "True" }
 subreddit = { optional = false, regex = "[_0-9a-zA-Z]+$", nmin = 3, explanation = "what subreddit to pull posts from, the name of the sub, not the URL", example = "AskReddit", oob_error = "A subreddit name HAS to be between 3 and 20 characters" }
-post_id = { optional = true, default = "", regex = "^((?!://|://)[+a-zA-Z])*$", explanation = "Used if you want to use a specific post.", example = "urdtfx" }
+post_id = { optional = true, default = "", regex = "^((?!://|://)[+a-zA-Z0-9])*$", explanation = "Used if you want to use a specific post.", example = "urdtfx" }
```

why not just `\w`?

```suggestion
post_id = { optional = true, default = "", regex = "^((?!://|://)[+\w])*$", explanation = "Used if you want to use a specific post.", example = "urdtfx" }
```

---

`/w` would allow `_` as well, which cannot be part of a post_id

---

good point, what about this?

```suggestion
post_id = { optional = true, default = "", regex = "^((?!://|://)[+\w][^_])*$", explanation = "Used if you want to use a specific post.", example = "urdtfx" }
```

---

could work, but imho `a-zA-Z0-9` is much easier to read